salt/states/docker_network.py

```diff
@@ -110,7 +129,11 @@ def present(name, driver=None, containers=None):
     else:
         try:
             ret['changes']['created'] = __salt__['docker.create_network'](
-                name, driver=driver)
+                name,
+                driver=driver,
+                driver_opts=driver_opts,
+                check_duplicate=True)
```

Comment:
@ticosax got it, I've updated the PR. Please let me know if everything is ok now. Thanks.